databases/week2/p3_hyf_lesson2.sql

```diff
@@ -0,0 +1,28 @@
+USE hyf_lesson2;
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.*, user.email
+  FROM task
+  JOIN user_task ON task.id = user_task.task_id
+  JOIN user ON user_task.user_id = user.id
+  WHERE user.email LIKE'%@spotify.com';
+
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.*
+FROM task
+JOIN status ON task.status_id = status.id
+JOIN user_task ON task.id = user_task.task_id
+JOIN user ON user_task.user_id = user.id
+WHERE user.name = 'Donald Duck' AND status.name = 'Not started';
+
+-- Get all the tasks for 'Maryrose Meadows' that were created in september
+SELECT task.*
+FROM task
+JOIN user_task ON task.id = user_task.task_id
+JOIN user ON user_task.user_id = user.id
+WHERE user.name = 'Maryrose Meadows' AND month(created)=09;
+
+-- Find how many tasks where created in each month, e.g. how many tasks were created in october, how many tasks were created in november, etc. 
+SELECT COUNT(title) AS tasks, month(created) AS month_number
```

Comment:
Good with a minor thing - you probably want to use COUNT(id) since titles are not unique, so if two tasks have the same title they will only be counted once